src/examples/typography/code/Color.tsx

```diff
@@ -8,27 +8,27 @@
 import React from 'react';
 import styled from 'styled-components';
 import { mediaQuery } from '@zendeskgarden/react-theming';
-import { Row, Col } from '@zendeskgarden/react-grid';
+import { Grid } from '@zendeskgarden/react-grid';
 import { Code } from '@zendeskgarden/react-typography';
 
-const StyledCol = styled(Col)`
+const StyledCol = styled(Grid.Col)`
   ${p => mediaQuery('down', 'xs', p.theme)} {
     margin-top: ${p => p.theme.space.sm};
   }
 `;
 
 const Example = () => (
-  <Row>
-    <Col sm={4} textAlign="center">
+  <Grid.Row>
+    <Grid.Col sm={4} textAlign="center">
       <Code hue="green">Veggies es bonus</Code>
-    </Col>
+    </Grid.Col>
     <StyledCol sm={4} textAlign="center">
       <Code hue="yellow">Veggies es bonus</Code>
     </StyledCol>
     <StyledCol sm={4} textAlign="center">
       <Code hue="red">Veggies es bonus</Code>
     </StyledCol>
```

Comment:
Making a mental note to update docs when `props.hue` supports variables.

ex: `<Span hue="foreground.subtle">`

Reply:
> Making a mental note to update docs when `props.hue` supports variables.
> 
> ex: `<Span hue="foreground.subtle">`

In this case, light/dark mode will provide the desired treatment for the hues provided. No change needed.